glide.lock

```diff
@@ -203,7 +203,7 @@ imports:
   - thrift-gen/zipkincore
   - utils
 - name: github.com/uber/jaeger-lib
-  version: 3b2a9ad2a045881ab7a0f81d465be54c8292ee4f
+  version: c48167d9cae5887393dd5e61efd06a4a48b7fbb3
```

Comment:
Do we need to update github.com/uber/jaeger-lib ?

Reply:
There is no difference in code between the two git version.

Reply:
I checked again, and there is code difference.